packages/replayio/src/utils/recordings/upload/uploadSourceMaps.ts

```diff
@@ -25,10 +25,13 @@ async function ensureResource(client: ProtocolClient, content: string) {
   return (await createResource(client, { content })).resource;
 }
 
+const queue = createPromiseQueue({ concurrency: 10 });
```

Comment:
This could live even closer to the `client` but I stayed away from doing any changes to that right now.